run/hello-broken/README.md

```diff
@@ -0,0 +1,47 @@
+# Cloud Run Broken Sample
+
+This sample presents broken code in need of troubleshooting. An alternate resource at /improved shows a more stable implementation with more informative errors and default values.
+
+Use it with the [Local Container Troubleshooting tutorial](http://cloud.google.com/run/docs/tutorials/local-troubleshooting).
```

Comment:
- I don't know exactly what we mean here. What is _it_? How do we use it?
  - Maybe say: "Troubleshoot this code by following the [Local Container Troubleshooting Tutorial](https://cloud.google.com/run/docs/tutorials/local-troubleshooting)"
- Nit: Why is "Tutorial" not capitalized in this page title too?

Reply:
Done.